paasta_tools/cli/cmds/spark_run.py

```diff
@@ -560,13 +543,12 @@ def get_spark_env(
     """Create the env config dict to configure on the docker container"""
 
     spark_env = {}
-    if not args.disable_aws_credential_env_variables:
-        access_key, secret_key, session_token = aws_creds
-        if access_key:
-            spark_env["AWS_ACCESS_KEY_ID"] = access_key
-            spark_env["AWS_SECRET_ACCESS_KEY"] = secret_key
-            if session_token is not None:
-                spark_env["AWS_SESSION_TOKEN"] = session_token
+    access_key, secret_key, session_token = aws_creds
+    if access_key:
+        spark_env["AWS_ACCESS_KEY_ID"] = access_key
+        spark_env["AWS_SECRET_ACCESS_KEY"] = secret_key
+        if session_token is not None:
+            spark_env["AWS_SESSION_TOKEN"] = session_token
```

Comment:
i'm looking forward to the drivers on k8s so that we can stop leaking these in the kubectl output for these pods :p